src/main/java/de/orchound/gameoflife/model/Rule.java

```diff
@@ -0,0 +1,41 @@
+package de.orchound.gameoflife.model;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Rule {
+	List<Integer> birth = new ArrayList<>();
+	List<Integer> survival = new ArrayList<>();
+
+	public Rule(String ruleString) {
+		parseRulestring(ruleString);
+	}
+
+	public boolean determineCellStatus(boolean cell, int livingNeighborsCount) {
+
+		if (cell) {
+			for (int x : survival) {
+				if (livingNeighborsCount == x)
+					return true;
+			}
+		} else {
+			for (int x: birth) {
+				if (livingNeighborsCount == x)
+					return true;
+			}
+		}
+		return false;
```

Comment:
```suggestion
	return cell ? survival.contains(livingNeighborsCount) : birth.contains(livingNeighborsCount);
```
I also suggest using sets for `birth` and `survival`.